subservers/manager.go

```diff
@@ -126,11 +130,19 @@ func (s *Manager) StartIntegratedServers(lndClient lnrpc.LightningClient,
 		)
 		if err != nil {
 			s.statusServer.SetErrored(ss.Name(), err.Error())
+			errs = append(errs, fmt.Sprintf("%s: %v", ss.Name(), err))
```

Comment:
I think we want to return an error imminently here, and not proceed to start all sub-servers before we return the error. The motivation for that is that we want `lnd` to shutdown asap if a "critical" sub-server errors during startup, and that could be delayed if we proceed to start the rest of the sub-servers before returning the error, and hence proceed to shutdown `lnd`.